### ISSUE TITLE
Are "multiple server URLs" only available in 1.4 branch ?

### ISSUE THREAD

**Issue by mpfj** [open62541/open62541#6356]
## Description
I see that the master/1.4 branch allows multiple server URLs to be defined using a string array.
(as explained in section 7.1.6 of the development docs)

But is such a feature available in the 1.3 branch ?

## Background Information / Reproduction Steps
I current set the server config using:-

```
UA_String sIP = UA_String_fromChars("10.191.1.123");
UA_String_clear(&config->customHostname);
UA_String_copy(&sIP, &config->customHostname);
UA_String_delete(&sIP);
```

But our product has 2 network ports so I'd like to be able to have the server accept connections on either IP address range.
For example, I'd like to be able to have the server listen on `ocp.tcp://10.191.1.123:4840` **and** `ocp.tcp://10.250.250.250:4840`

I have had a look as best I can but can't see a method of doing that.
If it is possible, can someone point me in the right direction?

## Checklist

Please provide the following information:

 - [x] open62541 Version (release number or git tag): 1.3.7
 - [ ] Other OPC UA SDKs used (client or server): 
 - [x] Operating system: Linux
 - [ ] Logs (with `UA_LOGLEVEL` set as low as necessary) attached
 - [ ] Wireshark network dump attached
 - [ ] Self-contained code example attached
 - [ ] Critical issue

**Comment by jpfr** [comment]
This is only available for 1.4.
The networking subsystem was greatly improved (EventLoop) and now has the flexibility to explicitly configure parallel network interfaces.

**Comment by mpfj** [comment]
Sorry to bring this issue back up but I'm back looking at getting "dual interfaces" working on v1.3

I have tried adding two separate servers (each one having its own IP address) but only the primary interface has an active listener.
Using netstat, I can see that nothing is listening on the second IP address.

So I then tried configuring a single server but with the secondary IP address ... and this didn't work either??

Does the IP address to bind to not get configured as per by original code snippet above?
If not, can someone explain how to correctly specify the IP address of the server?

Or is running two servers just not supported?

Thx
